presto-main/src/main/java/com/facebook/presto/operator/SimplePagesHashStrategy.java

```diff
@@ -209,4 +210,10 @@ public boolean isPositionNull(int blockIndex, int blockPosition)
         }
         return false;
     }
+
+    @Override
+    public int compare(int leftBlockIndex, int leftBlockPosition, int rightBlockIndex, int rightBlockPosition)
+    {
+        throw new NotImplementedException();
```

Comment:
At a time it wasn't crucial for performance tests and validation whether whole idea has any sense. I have added it now though.